src/plugins/mf_priority.cpp

```diff
@@ -30,9 +30,9 @@ extern "C" {
 #include <sstream>
 
 #define BANK_INFO_MISSING -9
-#define NO_SUCH_QUEUE -5
+#define UNKNOWN_QUEUE 0
+#define NO_QUEUE_SPECIFIED 0
```

Comment:
Commit message nit: The commit message says:
> Change the preprocessor directive INVALID_QUEUE to UNKNOWN_QUEUE

But it _appears_ that here `INVALID_QUEUE` is still defined.

Also:

> Change the preprocessor directive NO_DEFAULT_QUEUE to UNSPECIFIED_QUEUE,

I don't see a `UNSPECIFIED_QUEUE`, but there is a `NO_QUEUE_SPECIFIED`...

Reply:
Whoops, I totally messed that up, didn't I? I meant to update this commit message before posting the PR but completely forgot. I'll update the commit message to use both `UNKNOWN_QUEUE` (instead of `INVALID_QUEUE`) and `NO_QUEUE_SPECIFIED` (instead of `UNSPECIFIED_QUEUE`). Sorry for the confusion!

Reply:
While you are in there, it might be good to add comments on each of these #defines to say more of what each means. Not critical, but if you are modifying the commit anwyay...

Reply:
Yes, good suggestion. I will do that!